usr/src/cmd/zfs/zfs_main.c

```diff
@@ -2330,6 +2331,9 @@ us_compare(const void *larg, const void *rarg, void *unused)
 			if (rv64 != lv64)
 				rc = (rv64 < lv64) ? 1 : -1;
 			break;
+
+		default:
+			break;
```

Comment:
with 'default' or without 'default' - we will be outside of switch.
i don't know about - do we need assert or not - because for now - we are ignoring other properties.